tests/unit/frameworks/vernon/self_supervised_learning_test.py

```diff
@@ -108,6 +111,9 @@ def test_self_supervised_experiment(self):
         # Setup experiment and initialize model.
         exp = self_supervised_config["experiment_class"]()
         exp.setup_experiment(self_supervised_config)
+        assert isinstance(exp.model, EncoderClassifier)
+        assert hasattr(exp.model, "classifier")
+        assert hasattr(exp.model, "encoder")
         # Loop through some pseudo epochs.
         for _ in range(5):
             ret = exp.run_epoch()
```

Comment:
Unit tests should only report errors, there is no need to `print(ret)` here.

Reply:
removed the print statement